src/main/modules/mpris.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * Module in charge of setting and manage mpris service for linux
+   TODO: fetch cover, handle seek bar, handle volume    
+ */
+import * as electron from 'electron';
+import ModuleWindow from './module-window';
+import { TrackModel } from '../../shared/types/interfaces';
+import { fetchCover } from '../../shared/utils';
+import * as mpris  from 'mpris-service';
+const { app, ipcMain } = electron;
+
+class MprisModule extends ModuleWindow {
+
+  player = mpris({
+      name: 'museeks',
+      identity: 'Musseks Music Player',
+      canRaise: true,
+      supportedUriSchemes: ['file'],
+      supportedMimeTypes: ['audio/mpeg', 'application/ogg'],
```

Comment:
Here I think you should import the `SUPPORTED_TRACKS_EXTENSIONS` variables from `ui/lib/app.ts`.

Then you can simply map it with a module like `mime-type` https://www.npmjs.com/package/mime-types

So it looks like:

```js
import * as mime from 'mime-types';

...
supportedMimeTypes: SUPPORTED_TRACKS_EXTENSIONS.map(mime.lookup);
```

Reply:
ok, I'll add this later. Let's focus on the fetch cover feature.